### PR TITLE
Added Christmas themed Hubber

### PR DESCRIPTION
**Pull request by avneesh0612**
I have added a new Hubber, which is Christmas-themed!

### CODE REVIEW

**Review by krishguptadev — CHANGES_REQUESTED** [review]
A file should be added, rather than a link, add in `community/`

**Comment by avneesh0612** [comment]
oh okay

**Comment by avneesh0612** [comment]
Good now? @krishguptadev

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ Assets for EddieHub branding repo.
 ![20](https://user-images.githubusercontent.com/624760/114314292-f00b4b80-9af1-11eb-91c2-ca99a207343d.png)
 ![21](https://user-images.githubusercontent.com/624760/114314293-f0a3e200-9af1-11eb-978a-60a76155239a.png)
 ![22](https://user-images.githubusercontent.com/624760/114314294-f0a3e200-9af1-11eb-9019-b96b5b15cb7c.png)
+![23](https://user-images.githubusercontent.com/76690419/144421989-00568b89-37c8-4310-9ed9-a2dc89e6ed9e.png)
```
Comment (krishguptadev) [inline README.md:35]: This shouldn't be here afaik

Comment (avneesh0612) [reply README.md:35]: hmm, but all images should be in readme, right?

Comment (krishguptadev) [reply README.md:35]: the community-made ones are not there tho

Comment (eddiejaoude) [reply README.md:35]: Good spot Krish 👍 the `README.md` is only for the official images

A `community.md` could be added though 👍

Comment (avneesh0612) [reply README.md:35]: Oh okay cool

**Comment by avneesh0612** [comment]
Removed them 👍

**Review by krishguptadev — APPROVED** [review]
Awesome!

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**